.github/workflows/apply-milestone.yml

```diff
@@ -0,0 +1,63 @@
+name: Milestone Linker
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize, closed] # Trigger on relevant PR events
+    paths: # targeting only code that touches the app
+      - VAMobile/src/**
+
+jobs:
+  process_pr_and_issue:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+      issues: write
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Extract Linked Issue Number from PR Body
+        id: extract_issue
+        run: |
+          PR_BODY="${{ github.event.pull_request.body }}"
```

Comment:
I think `nearform-actions/github-action-check-linked-issues@v1`will return a list of the linked issues including the ones from the sidebar

Reply:
Also occurred to me that not all issues in the body will be "linked" issues